docs/tianocore_build.md

```diff
@@ -0,0 +1,89 @@
+# Coreboot with tianocore payload on apu2
+
+This documentation describes how to build coreboot image with tianocore payload
+for PC Engines apu2 platform.
+
+## Building coreboot image
+
+1. Download coreboot-sdk docker image
+```
+docker pull coreboot/coreboot-sdk:1.52
+```
+
+2. Run docker container with passing coreboot project directory
+```
+docker run --rm -it -v <coreboot-directory>:/home/coreboot coreboot/coreboot-sdk:1.52 /bin/bash
+```
+
+  Notice that `<coreboot-directory>` is your local directory with coreboot
+  project. `/home/coreboot` will be directory for that project in docker
+  container. Therefore, it's up to you what path to give.  
+
+3. In docker container go to coreboot main folder
+```
+coreboot@32ceb7327125:/$ cd home/coreboot
```

Comment:
remove the `coreboot@32ceb7327125:/`. The `$` is enough to indicate this is a terminal